hazelcast-integration/kubernetes/samples/wan/publisher/hazelcast.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config/hazelcast-config-4.0.xsd"
-           xmlns="http://www.hazelcast.com/schema/config"
-           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<hazelcast xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
This is the only k8s sample config that I've upped to 4.0 but not sure how to verify it. The `wan-replication` needs to be fixed if we are ok with 4.0.